blueocean-dashboard/src/main/resources/jenkins/plugins/blueocean/dashboard/Messages_zh.properties

```diff
@@ -0,0 +1,286 @@
+# common
+common.date.duration.format=d[d] h[h] m[m] s[s]
+common.date.duration.hint.format=M [month], d [days], h[ hours], m[ minutes], s[ seconds]
+common.date.duration.display.format=d[d] h[h] m[m] s[s]
+common.date.readable.long=MMM DD YYYY h:mma Z
+common.date.readable.short=MMM DD h:mma Z
+#加载中...
+common.pager.loading=\u52a0\u8f7d\u4e2d\u002e\u002e\u002e
```

Comment:
@gbraad Gerard, so the Chinese start with # is exactly the same as the unicode characters  on the right of the "=". 

I should had used converter to check the unicode first. Then I would know they are the same thing earlier.

OK, let's just focus on the Chinese strings begins with the hash sign.